### PR TITLE
Updates proj to use pipelines.

### PR DESCRIPTION
**Pull request by jlaura**
Updates to use the proper proj pipelines.

### CODE REVIEW

File: knoten/csm.py [hunk 1]
```diff
@@ -245,7 +252,7 @@ def generate_latlon_boundary(camera, boundary, dem=0.0, radii=None, **kwargs):
 
         gnds[i] = [gnd.x, gnd.y, gnd.z]
 
-    lons, lats, alts = pyproj.transform(ecef, lla, gnds[:,0], gnds[:,1], gnds[:,2])
+    lons, lats, alts = transformer.transform(ecef, lla, gnds[:,0], gnds[:,1], gnds[:,2])
```
Comment (AustinSanders) [inline knoten/csm.py]: Where do ecef and lla come from?

Comment (jlaura) [reply knoten/csm.py]: Good catch. Those need to be removed. Commit incoming.

**Comment by jlaura** [comment]
Also, opened #113. I went to update the test that should have caught this and find that csm.py is completely untested 😲

**Review by AustinSanders — APPROVED** [review]
I think this is good to go now!

**Pull request merged**